app/controllers/stories_controller.rb

```diff
@@ -38,10 +38,11 @@ def show
       handle_article_show
     elsif (@article = Article.find_by(slug: params[:slug])&.decorate)
       handle_possible_redirect
-    else
-      @podcast = Podcast.available.find_by!(slug: params[:username])
+    elsif (@podcast = Podcast.available.find_by(slug: params[:username]))
       @episode = @podcast.podcast_episodes.available.find_by!(slug: params[:slug])
       handle_podcast_show
+    else
+      not_found
```

Comment:
If something weird happens with a URL or `slug`, we can end up routed to the `StoriesController`, where this `if/elsif` cascades down through the various objects a `slug` might be. I'm not sure it's a **problem** to have "couldn't find this podcast" errors... but since we have `not_found`, anyway, it seems like we might as well use it? ¯\_(ツ)_/¯

Reply:
Displaying "Podcast not found" when it was actually "not found" was annoying :+1: